docs/client/javascript-mono/MigrationFromOldJsClient.mdx

```diff
@@ -76,12 +76,9 @@ Similar to initialization, the `updateUser` method now supports both synchronous
 ```typescript
 import Statsig from "statsig-js";
 
-// initialize returns a promise which always resolves
-await Statsig.initialize(
-  "client-sdk-key",
-  { userID: "some_user_id" },
-  { environment: { tier: "staging" } } // optional, pass options here if needed
-);
+const user = { userID: "a-user" };
+
```

Comment:
should note that this wont wait on network values?  Thats a breaking change in expectation over the existing sdk

Reply:
This is just illustrating the old way. I'll add an `await`